src/main/java/com/cleanroommc/groovyscript/helper/ingredient/IngredientHelper.java

```diff
@@ -25,6 +27,10 @@ public class IngredientHelper {
     public static final Closure<Object> NO_RETURN = new LambdaClosure<>(args -> ItemStack.EMPTY);
     public static final Closure<Object> MATCH_NBT = new LambdaClosure<>(args -> ItemStack.EMPTY);
 
+    public static ItemStack damageItem(ItemStack stack, int damage) {
+        return ItemStackMixinExpansion.of(stack).withMeta(Math.min(32767, Items.DIAMOND.getDamage(stack) + damage));
```

Comment:
could this method get some javadocs to explain this, particularly the `Items.DIAMOND` part? might be worth doing `Short.MAX_VALUE` instead of `32767`.